app/com/gu/floodgate/DynamoDBTable.scala

```diff
@@ -71,6 +84,20 @@ trait DynamoDBTable[T] extends StrictLogging {
     getListOfItems(request)
   }
 
+  def getLatestItem(id: String, filterKeyName: String, filterValue: String)(implicit ec: ExecutionContext): Future[Option[T]] = {
+    val request = new QueryRequest().withTableName(tableName)
+      .withScanIndexForward(false)
+      .withKeyConditionExpression(s"$keyName = :$keyName")
```

Comment:
@regiskuckaertz the IDs are not unique values within the DB tables, so the key is made up of an ID & an environment. Here the environment is the filterKey value. From what I understand, we need both `key` & `filterKey` in order to retrieve a specific item.